shell/platform/linux/fl_renderer.cc

```diff
@@ -443,13 +448,21 @@ void fl_renderer_setup(FlRenderer* self) {
   }
 }
 
-void fl_renderer_render(FlRenderer* self, int width, int height) {
+void fl_renderer_render(FlRenderer* self,
+                        int width,
+                        int height,
+                        GdkRGBA* background_color) {
   FlRendererPrivate* priv = reinterpret_cast<FlRendererPrivate*>(
       fl_renderer_get_instance_private(self));
 
   g_return_if_fail(FL_IS_RENDERER(self));
 
-  glClearColor(0.0, 0.0, 0.0, 1.0);
+  if (background_color != nullptr) {
+    glClearColor(background_color->red, background_color->green,
+                 background_color->blue, background_color->alpha);
+  } else {
```

Comment:
Maybe just to be super clear that this is what we intend:
```suggestion
  } else {
    // Set default view background color to black.
```